src/helpers/authentication.ts

```diff
@@ -0,0 +1,22 @@
+import { SignIn } from 'types/signIn';
+
+export const authTokenKey = 'AuthToken';
+
+export const getToken = () => {
```

Comment:
As we are getting a set of token
```suggestion
export const getTokens = () => {
```